CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # [Versions](https://mui.com/versions/)
 
+## v5.16.2
+
+<!-- generated comparing v5.16.1..master -->
+
+_Jul 16, 2024_
+
+A big thanks to the 2 contributors who made this release possible. Here are some highlights ✨:
+
+- `@mui/base` is no longer a dependency of `@mui/material`
```

Comment:
```suggestion
A big thanks to the 2 contributors who made this release possible.
```
I wouldn't highlight this, it's more of a core change, it doesn't affect end-users.